src/main/docs/serializers.adoc

```diff
@@ -0,0 +1,265 @@
+ifndef::sourcedir[]
+:sourcedir: ../..
+endif::sourcedir[]
+
+= Custom Serializers
+
+The Ehcache 3 documentation at http://www.ehcache.org/documentation/3.0/serializers-copiers.html#serializers[Serializers]
+gives you an overview of how to use custom serializers with a cache.
+The section on http://www.ehcache.org/documentation/3.0/serializers-copiers.html#persistent-vs-transient-caches[Persistent and Transient Serializers]
+briefly covers the serializer contracts that must be honored while writing custom serializers to be used with 
+persistent/transient caches.
+
+This document explains how you can write a transient/persistent custom serilaizer that works with Ehcache.
+Here we discuss the significance of transient serializers and persistent serializers in detail and what they actually 
+mean.
+
+== Serializer types
+
+In the Ehcache getting started document you must have already seen the serializers requiring a single argument constructor
```

Comment:
I would not assume this knowledge.
Instead: "As indicated in the Ehcache documentation, serialisers require ..."